surveyor.py

```diff
@@ -397,7 +411,7 @@ def survey(ctx, product_str: str = 'cbr') -> None:
 
                 ioc_list = [x.strip() for x in data]
 
-                product.nested_process_search(Tag(f"IOC - {opt.ioc_file}", data=basename), {opt.ioc_type: ioc_list}, base_query)
+                product.nested_process_search(Tag(f"IOC - {opt.ioc_file}", data=basename), {opt.ioc_type: ioc_list}, base_query, opt.json)
```

Comment:
`opt.json` can be passed when initializing the product

```suggestion
                product.nested_process_search(Tag(f"IOC - {opt.ioc_file}", data=basename), {opt.ioc_type: ioc_list}, base_query)
```